app/views/submissions/_assigned_evaluators.html.erb

```diff
@@ -50,18 +50,20 @@
                   <td data-label="">
                     <div class="display-flex flex-wrap">
                       <% if assignment&.evaluation&.completed_at %>
-                        <%= link_to(evaluation_path(assignment.evaluation)) do %>
+                        <%= link_to(edit_evaluation_path(assignment.evaluation)) do %>
```

Comment:
this will be updated to a new route later for CM overriding the complete evaluation
```suggestion
                        <%= link_to(evaluation_path(assignment.evaluation)) do %>
```